cosmos/providers/dbt/core/operators.py

```diff
@@ -79,6 +75,10 @@ class DbtBaseOperator(BaseOperator):
     :param dbt_executable_path: Path to dbt executable can be used with venv
         (i.e. /home/astro/.pyenv/versions/dbt_venv/bin/dbt)
     :type dbt_executable_path: str
+    :param dbt_cmd_flags: Flags passed to dbt command override those that are calculated.
+    :type dbt_cmd_flags: dict
+    :param profiles_dir: Which directory to look in for the profiles.yml file. Default is ~/.dbt/profiles.yml.
```

Comment:
We have actually changed the profiles through setting an environment variable, so we are not allowing people to set the profile via an argument. Can you please remove this option?

The reason we do this is because we need to force the `profiles.yaml` to be in the base `~/.dbt` directory for our "push airflow connections into dbt profiles" magic to work

You can read a bit more here https://astronomer.github.io/astronomer-cosmos/dbt/connections.html

Reply:
Done